layouts/_default/_markup/render-codeblock-mermaid.html

```diff
@@ -0,0 +1,12 @@
+<!-- prettier-ignore-start -->
+{{ if not (.Page.Scratch.Get "mermaid") }}
+  <!-- Include mermaid only first time -->
+  <script defer src="{{ index (index .Site.Data.assets "mermaid.js") "src" | relURL }}"></script>
+  {{ .Page.Scratch.Set "mermaid" true }}
+{{ end }}
+<!-- prettier-ignore-end -->
+
+{{ .Page.Scratch.Set "mermaid" true }}
+<pre class="gdoc-mermaid mermaid">
+  {{- .Inner -}}
+</pre>
```

Comment:
Is there somewhere in the docs to specify a minimum Hugo version? This requires v0.93.0 or later, which was only released at the end of February 2022.

Reply:
Yes there are three files to change:

- [README](https://github.com/thegeeklab/hugo-geekdoc/blob/main/README.md) Line 4
- https://github.com/thegeeklab/hugo-geekdoc/blob/main/exampleSite/content/en/_index.md Line 12
- https://github.com/thegeeklab/hugo-geekdoc/blob/main/theme.toml#L8